src/background/hypothesis-chrome-extension.js

```diff
@@ -150,20 +154,24 @@ export default function HypothesisChromeExtension({
   // exposed for use by tests
   this._onTabStateChange = onTabStateChange;
 
+  /** @param {chrome.tabs.Tab} tab */
   function onBrowserActionClicked(tab) {
-    const tabError = state.getState(tab.id).error;
+    const tabId = /** @type {number} */ (tab.id);
```

Comment:
I am assuming in several places in this PR that tabs have an ID and URL. We've never seen any reports of errors due to this not being the case, though I couldn't completely guarantee that it has never happened.